### PR TITLE
FAVCS-64: Can not change name on in a contact box

### PR DESCRIPTION
**Pull request by mnakov**
### Changes
- Unhide hidden field "Name (single line)" to allow it's edition
  
### Technical Details
- Exported content_type_phone_book feature from live
- Unhide "name_line" in " field_phone_book_address" in content type "phone_book"
- Recreate content_type_phone_book feature
- Added hook update for reverting content_type_phone_book feature
  
### Affected Pages
- content type "phone_book"
  
### Key Affected Code
- N/A
  
### Key Functionality in custom code
- N/A
  
### References
- N/A
  
### Notices
- drush updb

### CODE REVIEW

File: docroot/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install [hunk 1]
```diff
@@ -1002,3 +1002,11 @@ function favrskov_helper_update_7054() {
 function favrskov_helper_update_7055() {
   features_revert_module('media_imagestyles');
 }
+
```
Comment (lachezarvalchev) [inline docroot/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install]: @mnakov and update hook like this one is only ok, if we are 100% sure that we have the latest feature exported from live before the deploy

File: docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc [hunk 2]
```diff
@@ -19,11 +20,11 @@ function content_type_phone_book_ctools_plugin_api($module = NULL, $api = NULL)
 function content_type_phone_book_node_info() {
   $items = array(
     'phone_book' => array(
-      'name' => t('Contact item'),
```
Comment (lachezarvalchev) [inline docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc]: It is not ok to have Danish strings in the t() function. It should be the English string and then to do the translation manually via the Translation UI after deploy

File: docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc [hunk 3]
```diff
@@ -19,11 +20,11 @@ function content_type_phone_book_ctools_plugin_api($module = NULL, $api = NULL)
...
+      'name' => t('Kontaktoplysning'),
       'base' => 'node_content',
       'description' => '',
       'has_title' => '1',
-      'title_label' => t('Title'),
```
Comment (lachezarvalchev) [inline docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc]: Same as the other one :)

File: docroot/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install (hunk 1 above)
Comment (mnakov) [reply docroot/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install]: @lachezarvalchev, I confirm that the first thing that I did was to export the feature from live.

File: docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc (hunk 2 above)
Comment (mnakov) [reply docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc]: @lachezarvalchev, fixed.

File: docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc (hunk 3 above)
Comment (mnakov) [reply docroot/profiles/favrskov/modules/features/content_types/content_type_phone_book/content_type_phone_book.features.inc]: @lachezarvalchev, fixed.

**Pull request merged**